src/extension.ts

```diff
@@ -822,13 +822,45 @@ export async function activate(context: vscode.ExtensionContext): Promise<any> {
     vscode.commands.registerCommand("vscode-objectscript.pickProcess", async (config) => {
       const system = config.system;
       const api = new AtelierAPI(vscode.window.activeTextEditor?.document.uri);
-      const convert = (data) =>
-        data.result.content.map(
-          (process: AtelierJob): vscode.QuickPickItem => ({
-            label: process.pid.toString(),
-            description: `Namespace: ${process.namespace}, Routine: ${process.routine}`,
-          })
-        );
+      const convert = async (jobData) => {
+        // NOTE: We do not know if the current user has permissions to other namespaces, so lets only fetch the job infos
+        // for the current namespace.
+        const currNamespaceJobs: { [k: string]: string } = await api
+          .actionQuery("SELECT Job, ConfigName FROM Ens.Job_Enumerate() where State = 'Alive'", [])
+          .then((data) => Object.fromEntries(data.result.content.map((x) => [x.Job, x.ConfigName])))
+          .catch((error) => {
+            // Current namespace is not Interoperability-enabled, there is no Ens.Job_Enumerate procedure
+            if (error && error.errorText.includes("Table valued function 'ENS.JOB_ENUMERATE'(...) not found")) {
```

Comment:
Does error.errorText get localised? In case it does, maybe just check it includes the function name.

```suggestion
            if (error && error.errorText.includes("'ENS.JOB_ENUMERATE'(...)")) {
```